game/views.py

```
@@ -572,3 +573,35 @@ def stats_view(request):
         'ai_wins': ai_wins,
         'ai_draws': ai_draws,
     })
+
+
+@require_GET
+def opening_lookup(request):
+    """Return the chess opening matching the given FEN position.
+
+    Query params:
+        fen – a 3-field FEN key (board / side / castling).
```

Ruff (0.15.12)

[warning] Docstring contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF002)